src/Symfony/Component/HttpFoundation/Exception/InvalidHostException.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
```

Comment:
Which interface would be appropriate in this case?
For example currently the class throws exceptions of the class [ConflictingHeadersException]( https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Exception/ConflictingHeadersException.php) but that doesn't implement any interface.